ui/admin/app/routes/scopes/scope/authenticate/method.js

```diff
@@ -36,13 +37,11 @@ export default class ScopesScopeAuthenticateMethodRoute extends Route {
   }
 
   /**
-   * Opens the specified URL in a new tab or window.  By default this uses
-   * `window.open`, but may be overriden.
+   * Opens the specified URL in a new tab or window.
    * @param {string} url
    */
-  async openExternalOIDCFlow(url) {
-    // TODO don't use window directly
-    await window.open(url);
+  openExternalOIDCFlow(url) {
+    this.window.open(url);
```

Comment:
Should this be using newly introduced service?

Reply:
It will in the next PR.  I wanted to introduce the window manager service before usage, since there are some possible refactors that could be made to support it.

Reply:
Gotcha. Ty!